docs/readme.html

```diff
@@ -243,8 +245,6 @@
   pc.setLocalDescription(myOffer);
   ```
 
-  _setLocalDescription_ indirectly controls the candidate gathering process.
```

Comment:
AFAIK The gathering can start before setLocalDescription as an optimization to make the establishment faster:
https://tools.ietf.org/html/draft-ietf-rtcweb-jsep-19#section-3.5.4